.travis.yml

```diff
@@ -15,20 +15,20 @@ refs:
       - 'mkdir -p "$HOME/.prebuilt"'
       - 'ls "$HOME/.prebuilt"'
       - 'rm -rf "$HOME/.prebuilt"/*.jar'
-      - 'cp target/renaissance-gpl-0.9.0.jar "$HOME/.prebuilt/"`git rev-parse HEAD`.jar'
```

Comment:
Opt: perhaps there is a way to add an env variable to this file, to have a single version declaration.
If you do, I would suggest adding a comment to `version.sbt`, with a reminder that `.travis.yml` should also be changed when changing `version.sbt`.

Reply:
That's a good idea. I'll do it.
Let's try to make the release process as easy as possible. Btw, coffee is on me for the one who will take care of #46 ;)